### DISCUSSION TITLE
Is it possible to detach MJ SD01 LED's from Dimmer?

### DISCUSSION THREAD

**Discussion by thermoscookies** [arendst/Tasmota#10818] [Q&A] [answered]
Reading through https://tasmota.github.io/docs/devices/Martin-Jerry-MJ-SD01/
I was curious if it might be possible to control the dimmer LED's without actually dimming the light. I want full power going to the light but I use the up/down buttons to dim something else but would love to be able to use the status lights to show the level of that.

**Comment by barbudor** [comment, marked as answer]
Do you mean leds on the unit connected to GPIOs 14, 12, 5 and 3 ?
You can use the command [LedPower](https://tasmota.github.io/docs/Commands/#ledpower)

**Comment by pcdiem** [comment]
If you're using a generic module, you can do whatever you want. The bottom LED is tied to the relay. The other LEDs don't do anything unless you tell them to with the LedPower command.

If you're using the PWM Dimmer module, the LEDs will turn on to indicate the current dimmer level. You can use the LedMask command to specify which LEDs, if any, the PWM Dimmer module uses to indicate the dimmer level. Any LEDs not set in the mask can be controlled with the LedPower command.

**Comment by thermoscookies** [comment]
Thank you for the response. It looks like I need to play with the LedMask command. I tried LedPower and, as expected, by default it turns on/off the power led on the bottom. My hope is to control the green led's on the side, being able to turn between 1 to all 5 on. 

From looking at the board they appear to be LED4.

I'll test out some LedMask bitmask's to see if I can figure this out. 

Thank you again for the input!

ledpower turned the bottom 'power status' led to red

![image](https://user-images.githubusercontent.com/22629136/106773508-fbc02a80-65f5-11eb-92bb-e053d0ec968e.png)
